auxiliary/conclusion.tex

```diff
@@ -8,18 +8,28 @@
 as your work requires you to become progressively more familiar
 with each of the topics included in the guide.
 
-We started the book with a discussion of research as a public service:
-one that requires you to be accountable to both research participants
-and research consumers.
-We then discussed the current research environment,
+We started the book with a discussion of 
+credibility, transparency, and reproducibility in research:
+an overarching idea that your work should always be
+accessible and available to others, both within and outside your team.
+We then discussed the current research work environment,
 which necessitates cooperation with a diverse group of collaborators
 using modern approaches to computing technology.
-We outlined common research methods in impact evaluation,
-with an eye toward structuring data work.
+We outlined methods for planning your data work
+and creating basic documentation for data,
+so that multiple data sources can be linked without error
+you can map the structure of your data to your research design.
```

Comment:
Something is off in this sentence. Is this what you meant?
```suggestion
We outlined methods for planning your data work
and creating basic documentation for data,
so that multiple data sources can be linked without error
and so that you can map the structure of your data
to your research design.
```